R/autoload.R

```diff
@@ -81,3 +81,24 @@ renv_autoload_impl <- function() {
   TRUE
 
 }
+
+# TODO: this gets really dicey once the user starts configuring where
+# renv places its project-local state ...
+renv_project_find <- function(project = NULL) {
```

Comment:
Why was this moved to `autoload.R`? By convention, files with the prefix `renv_<name>` and placed in `<name>.R`.

Reply:
I moved it because that was the only place it was used, but since I'm reverting that change, I'll move it back.